extension/otto/gohan_sync.go

```diff
@@ -136,6 +137,42 @@ func init() {
 
 				return otto.NullValue()
 			},
+			"gohan_sync_update": func(call otto.FunctionCall) otto.Value {
```

Comment:
I described why gohan_sync_update is included in this patch. Honestly, I wondered if I should create separate patch, but this code is basically copy-and-paste from gohan_sync_delete so I think it's not so important enough to create a separate PR, IMO. Of course, if other reviewers also requested, I can create a separate patch.